src/extension/importer/core/src/main/java/org/geoserver/importer/format/GeoJSONFormat.java

```diff
@@ -159,7 +159,7 @@ ImportTask task(ImportData data, Catalog catalog) throws IOException {
 
         // get the composite feature type
         SimpleFeatureType featureType = new FeatureJSON().readFeatureCollectionSchema(file.file(), false);
-        System.out.println(featureType);
+        LOG.log(Level.DEBUG, featureType);
```

Comment:
Travis reports a compile error here:

````
[ERROR] COMPILATION ERROR : 
[INFO] -------------------------------------------------------------
[ERROR] /home/travis/build/geoserver/geoserver/src/extension/importer/core/src/main/java/org/geoserver/importer/format/GeoJSONFormat.java:[162,21] error: cannot find symbol
[INFO] 1 error
````

Hum... I'd say the problem is that DEBUG is a Log4J level, but the API we are using is Java logging (even if it then delegates to Log4J under covers)